### ISSUE TITLE
The docs example for how tagged template literals work is wrong

### ISSUE THREAD

**Issue by slikts** [WebReflection/hyperHTML#275]
A fiddle with the example; the "same unique Array" assertion fails: https://jsfiddle.net/slikts/zbwyhs3a/

Example in question: https://viperhtml.js.org/hyperhtml/documentation/#introduction-1

The reason the arrays are not the same is that it also needs to be the same template literal. Namely, instead of:
```js
template`1 ${2} 3`; // 1 2 3
template`1 ${4} 3`; // 1 4 3
```
It should be wrapped in a function:
```
const foo = x => template`1 ${x} 3`;
foo(2);
foo(4);
```
Updated fiddle where the assertion works: https://jsfiddle.net/slikts/zbwyhs3a/5/

**Comment by WebReflection** [comment]
@slikts correct, you can read more in here why the example was like that:
https://itnext.io/a-tiny-disastrous-ecmascript-change-fadc05c83e69

TL;DR they changed specs recently and I haven't had the time to catch up with the docs, any help welcome, thanks

**Comment by spencerkordecki** [comment]
Using the example @slikts provided, I opened a PR on the documentation repo here:
 
https://github.com/viperHTML/viperhtml.github.io/pull/30

**Comment by WebReflection** [comment]
thanks @spencerkordecki !
